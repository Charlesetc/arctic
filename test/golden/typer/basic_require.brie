require required_file

define main [
  (required_file.example_function ()).x
]
