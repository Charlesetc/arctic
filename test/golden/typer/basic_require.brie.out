root[(ident(require):: ident(required_file)::)::, (ident(define):: ident(main):: block[][((ident(required_file)::<example_function = Function, main = Function>.example_function::Function ()::Unit)::<x = Integer>.x::Integer)::Integer]::Function)::]::